lib/settings/pages/general_settings_page.dart

```diff
@@ -662,7 +702,7 @@ class _GeneralSettingsPageState extends State<GeneralSettingsPage> with SingleTi
                   },
                   subtitle: enableInboxNotifications
                       ? !kIsWeb && Platform.isAndroid && areAndroidNotificationsAllowed == true
-                          ? l10n.backgroundCheckWarning
+                          ? null
```

Comment:
I took this out because this information is now covered in the dialog (and the experimental heading) and to keep the settings page more clean.